pages/apply.html

```diff
@@ -0,0 +1,380 @@
+
+          <select name="entry.227038756" id="entry_227038756" aria-label="What position and GS level are you applying to? GS level is the pay grade. Please see &quot;Government pay grades explained:&quot; https://pages.18f.gov/joining-18f/pay-grades/ " aria-required="true" required=>
+            <option value=""></option>
+            <option value="Product Manager (GS 14)">Product Manager (GS 14)</option>
+            <option value="Product Manager (GS 15)">Product Manager (GS 15)</option>
+          </select>
+
+        </div>
+
+
+      </fieldset>
+
+      
+      <fieldset>
+
+        <h2><legend>Where can we find you online?</legend></h2>
+
+        <p>Add up to three links (e.g. GitHub, LinkedIn, your portfolio, etc.).</p>
+
+        <div class="usa-input">
+          <label for="entry_665572666">Link 1</label>
+          <input type="url" name="entry.665572666" value="" class="ss-q-short" id="entry_665572666" dir="auto" aria-label="Link 1  Must be a valid URL" title="Must be a valid URL">
+        </div>
+
+        <div class="usa-input">
+          <label class="ss-q-item-label" for="entry_1176824627">Link 2</label>
+          <input type="url" name="entry.1176824627" value="" class="ss-q-short" id="entry_1176824627" dir="auto" aria-label="Link 2  Must be a valid URL" title="Must be a valid URL">
+        </div>
+
+        <div class="usa-input">
+          <label class="ss-q-item-label" for="entry_2076423552">Link 3</label>
+          <input type="url" name="entry.2076423552" value="" class="ss-q-short" id="entry_2076423552" dir="auto" aria-label="Link 3  Must be a valid URL" title="Must be a valid URL">
+        </div>
+
+      </fieldset>
+
+
+      <fieldset class="resume">
+
+        <h2><legend>Resume:<span class="required">*</span></legend></h2>
+
+        <div class="usa-input">
+
+          <p>For guidance on how to build a government-style resume, please visit this link: <a href="https://pages.18f.gov/joining-18f/government-style-resume-guide/">https://pages.18f.gov/joining-18f/government-style-resume-guide/</a></p>
+
+          <p>Please copy and paste your résumé (or a link to it) below. Do NOT include your Social Security Number. Please be aware that cutting and pasting will remove most formatting.</p>
+
+          <textarea name="entry.710467749" rows="8" cols="0" class="ss-q-long" id="entry_710467749" dir="auto" aria-label="Resume: Please copy and paste your résumé (or a link to it) below. Do NOT include your Social Security Number. Please be aware that cutting and pasting will remove most formatting." aria-required="true" required></textarea>
+
+        </div>
+
+      </fieldset>
+
+
+      <fieldset class="why">
+
+        <h2><legend>About the "why:"</legend></h2>
+
+        <div class="usa-input">
+
+          <label class="ss-q-item-label" for="entry_510169774">Why do you want to join 18F?<span class="required">*</span></label>
```

Comment:
Let's make these two fields required too. (This is controlled via the Google form admin interface.)